llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -430,7 +430,9 @@ void RISCVInstrInfo::copyPhysRegVector(
     if (UseVMV) {
       const MCInstrDesc &Desc = DefMBBI->getDesc();
       MIB.add(DefMBBI->getOperand(RISCVII::getVLOpNum(Desc)));  // AVL
-      MIB.add(DefMBBI->getOperand(RISCVII::getSEWOpNum(Desc))); // SEW
+      unsigned Log2SEW =
+          DefMBBI->getOperand(RISCVII::getSEWOpNum(Desc)).getImm();
+      MIB.addImm(Log2SEW ? Log2SEW : 3);                        // SEW
```

Comment:
Just to confirm I understand: before this we could set accidentally this `vmv.v.v` to `sew=1` (even if it was not used anywhere later) and now it gets normalised to `sew=8` in that case, right?

Reply:
That's correct.

Reply:
Actually never mind it's an immediate. We could maybe set SEW to something illegal to prevent anything from depending on it

Reply:
> Actually never mind it's an immediate. We could maybe set SEW to something illegal to prevent anything from depending on it

That would complicate the verifier.